apps/slinky-slurm-controlplane/defaults.yaml

```diff
@@ -320,26 +320,20 @@ compute:
       #
       # -- (object)
       # Set affinity for Kubernetes Pod scheduling.
-      affinity: {}
-        # nodeAffinity:
-        #   requiredDuringSchedulingIgnoredDuringExecution:
-        #     nodeSelectorTerms:
-        #       - matchExpressions:
-        #           - key: "kubernetes.io/os"
-        #             operator: In
-        #             values:
-        #               - linux
-        # podAntiAffinity:
-        #   requiredDuringSchedulingIgnoredDuringExecution:
-        #     - topologyKey: "kubernetes.io/hostname"
-        #       labelSelector:
-        #         matchExpressions:
-        #           - key: "app.kubernetes.io/name"
-        #             operator: In
-        #             values:
-        #               - slurmctld
-        #               - slurmdbd
-        #               - slurmrestd
+      affinity:
```

Comment:
we should add a comment here to describe that we are wanting to make sure slurm doesn't run on the nodes where juypterhub is currently using the node, at least I think that is what we are saying here.